web/webkit/src/main/scala/net/liftweb/http/LiftServlet.scala

```diff
@@ -114,14 +114,21 @@ class LiftServlet extends Loggable {
     }
 
     if (reqOrg.request.suspendResumeSupport_?) {
-      runFunction(liftResponse => {
-        // do the actual write on a separate thread
-        Schedule.schedule(() => {
-          reqOrg.request.resume(reqOrg, liftResponse)
-        }, 0.seconds)
-      })
+      try {
+        reqOrg.request.suspend(cometTimeout)
+      } finally {
+        // FIXME This should be removed from the finally block in the next major
+        // FIXME Lift version; it is currently here to ensure any code that
+        // FIXME depends on a response handler running even if requests suspension
+        // FIXME fails continues to work as-is during the 3.x series.
+        runFunction(liftResponse => {
```

Comment:
I don't think it's worth the log noise, personally. The exception will already show up in the logs unless someone's configured exceptions to be suppressed, so that's its own thing. I think clearly noting the change in behavior in a hypothetical future release is a better approach.